### PR TITLE
Updatet some line in Readme.md for better understanding

### PR DESCRIPTION
**Pull request by mahoffmann**
improved Readme for better understanding of git pull
mostly in line 237
@marcusfuchs

### CODE REVIEW

File: EBC-Git-101/README.md [hunk 1]
```diff
@@ -233,7 +233,8 @@ And of course, branch `issue1_text` has not been changed on the server repo. The
 
 ![33_master](https://cloud.githubusercontent.com/assets/5516900/16191065/b9cc9cfe-36e2-11e6-87ea-d8f3d675996a.gif)
 
-Now our local working copy is back on the master, enabling us to pull the server's master:
+Now our local working copy is back on the master, enabling us to pull the server's master.
+Do not get confused by git's answer: `your branch is up-to-date with 'origin/master'.`. Your local repository is not aware of the changes you just did on the server repo, so this is a comparison between your working copy (you just checked out) and the local master. Like for many other actions, it's highly recommended to start a `git pull` only with a clean working copy. This means that you should not have any uncommitted local changes before you pull.
```
Comment (marcusfuchs) [inline EBC-Git-101/README.md]: @mahoffmann 
Thanks for the suggestion, I think this is really an important point to add. I would agree to the first part in line 237 

>Do not get confused by git's answer: `your branch is up-to-date with 'origin/master'.`. Your local repository is not aware of the changes you just did on the server repo, so this is a comparison between your working copy (you just checked out) and the local master.

The second part is written in a broader sense and touches further topics:

>Like for many other actions, it's highly recommended to start a `git pull` only with a clean working copy. This means that you should not have any uncommitted local changes before you pull

I would suggest to stick to the original point and replace the second part with something like 

> To update your repo to the latest information on the server, use `git fetch origin`. If you follow this with a `git status` request, git should report with a new message informing you that your local copy is behind the server's version.

**Review by marcusfuchs — COMMENTED** [review]
@mahoffmann 
I added a line comment to suggest a slightly different approach to the topic. What do you think?

File: EBC-Git-101/README.md (hunk 1 above)
Comment (mahoffmann) [reply EBC-Git-101/README.md]: @marcusfuchs works for me... I just found the other text when approaching the topic but your version sticks more to the original issue, as you already mentioned.

**Comment by marcusfuchs** [comment]
@mahoffmann 
sorry to be a bit pedantic here, but I would suggest to write "git fetch origin" and "git status" in your recent addition as "`git fetch origin`" and "`git status`". Then I'd be happy to merge.

**Comment by marcusfuchs** [comment]
Perfect, thanks for the contribution!

**Comment by mahoffmann** [comment]
Then be happy..
sorry for the neglect

**Pull request merged**